README.md

```diff
@@ -38,16 +38,17 @@ With the Gemini CLI you can:
 
 You are now ready to use the Gemini CLI!
 
-### For advanced use or increased limits:
+### Use a Gemini API key:
 
-If you need to use a specific model or require a higher request capacity, you can use an API key:
+The Gemini API provides a free tier with [100 requets per day](https://ai.google.dev/gemini-api/docs/rate-limits#free-tier) using Gemini 2.5 Pro, control over which model you use, and access to higher rate limits (with a paid plan):
 
 1. Generate a key from [Google AI Studio](https://aistudio.google.com/apikey).
```

Comment:
Shall we update this since OAuth (Gmail/Workspace) user may not need to use the key?